app/uk/gov/hmrc/transitmovementsvalidator/services/JsonValidationService.scala

```diff
@@ -71,7 +72,7 @@ trait JsonValidationService {
 
 class JsonValidationServiceImpl @Inject() extends JsonValidationService {
 
-  private val mapper = new ObjectMapper
+  private val mapper = new ObjectMapper().enable(DeserializationFeature.FAIL_ON_TRAILING_TOKENS);
```

Comment:
nitpick: that semicolon doesn't need to be there.

Reply:
Removed